catkin_ws/src/image_rotation/src/image_rotation_subscriber.cpp

```diff
@@ -1,38 +1,41 @@
-#include <iostream>
-#include <ros/ros.h>
-#include "../include/utilities.hpp"
+///
+/// @file
+/// @brief Implementation of ROS Image Rotation Subscriber
+///
 
+#include <iostream>
 
-#include <image_transport/image_transport.h>
-#include "std_msgs/String.h"
-#include "sensor_msgs/Image.h"
-#include <sensor_msgs/image_encodings.h>
+#include "../include/image_rotation/image_algorithm.h"
 
 #include <cv_bridge/cv_bridge.h>
 #include <opencv2/imgproc/imgproc.hpp>
 #include <opencv2/highgui/highgui.hpp>
```

Comment:
Please check if both the headers of open cv are required .

Reply:
Both the headers are not required and therefore are removed